pycbc/inference/models/relbin.py

```diff
@@ -655,11 +655,23 @@ def extra_args_from_config(cp, section, skip_args=None, dtypes=None):
         )
 
         # get fiducial params from config
-        fid_params = {
-            p.replace("_ref", ""): float(cp.get("model", p))
-            for p in cp.options("model")
-            if p.endswith("_ref")
-        }
+        fid_params = {}
+        for p in cp.options("model"):
+            if p.endswith("_ref"):
+                val = cp.get("model", p)
+                if val == '':
```

Comment:
There's a couple places now where we try to do this str -> something python conversion. Maybe we should centralize this now?

Reply:
Well, if we add it to a parser, it should be the base InterpolatingConfigParser, but yeah, that approach makes sense, e.g. extend get_opt